highfidelity/hifi
# glTF import DEMO

This PR allows users to create and import new models using the glTF file format.

...

## TEST PLAN
- Go to the Sandbox or other domain where you can create entities.
- Create a new model with the url: https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/DamagedHelmet/glTF/DamagedHelmet.gltf
- The glTF model should appear in front of you.

...

## KNOWN ISSUES
- It will only import static geometry.
- No bone information or animations will be imported.
